bundles/core-combat/player-events.js

```diff
@@ -17,12 +17,12 @@ module.exports = (srcPath) => {
         // could potentially wind up in a situation where the player performed
         // a mid-round attack that killed the target, then the next round the
         // target kills the player. So let's not let that happen.
-        if (this.attributes.health <= 0) {
+        if (this.getAttribute('health') <= 0) {
           this.combatants.forEach(combatant => {
             this.removeCombatant(combatant);
             combatant.removeCombatant(this);
             if (!combatant.isInCombat()) {
-              combatant.attributes.health = combatant.attributes.maxHealth;
+              combatant.setAttribute('health', combatant.getRawAttribute('maxHealth'));
```

Comment:
> I would hope there would not be any effects that modify just maxHealth and not health as well.

I could imagine a skill that applied an effect that temporarily boosted max health and left health alone in which case when you use `getRawHealth` you'd be setting their health to something less than their effective max health.